src/test/style.ts

```diff
@@ -67,9 +66,9 @@ describe('style', function() {
     assert.equal(elm.firstChild.style.fontSize, '10px');
   });
   it('updates css variables', function() {
-    var vnode1 = h('div', {style: {'--myVar': 1}});
-    var vnode2 = h('div', {style: {'--myVar': 2}});
-    var vnode3 = h('div', {style: {'--myVar': 3}});
+    var vnode1 = h('div', {style: {'--myVar': 1 as any}});
+    var vnode2 = h('div', {style: {'--myVar': 2 as any}});
+    var vnode3 = h('div', {style: {'--myVar': 3 as any}});
     elm = patch(vnode0, vnode1).elm;
     assert.equal(elm.style.getPropertyValue('--myVar'), 1);
```

Comment:
I wonder why these even use numbers instead of strings.

What I find interesting is that they come out as numbers on the other side, as well.